tests/BaseKernelTestCase.php

```diff
@@ -148,7 +149,7 @@ protected function createTag(array $properties = []): Tag
 
     protected function createLink(array $properties = []): Link
     {
-        $link = new Link();
+        $link = new ExternalLink();
```

Comment:
It's better to create new fixtures for different features to avoid modifying existing tests